ui/webui/src/components/Error.jsx

```diff
@@ -30,7 +30,34 @@ import { exitGui } from "../helpers/exit.js";
 
 const _ = cockpit.gettext;
 
-export const CriticalError = ({ exception, isBootIso }) => {
+export const bugzillaPrefiledReportURL = (productQueryData) => {
+    const baseURL = "https://bugzilla.redhat.com";
+    const queryData = {
+        ...productQueryData,
+        component: "anaconda",
+    };
+
+    const reportURL = new URL(baseURL);
+    reportURL.pathname = "enter_bug.cgi";
+    Object.keys(queryData).map(query => reportURL.searchParams.append(query, queryData[query]));
+    return reportURL.href;
+};
+
+const addExceptionDataToReportURL = (url, exception) => {
+    const newUrl = new URL(url);
+    newUrl.searchParams.append(
+        "short_desc",
+        "WebUI: " + exception.message
```

Comment:
I would suggest we also include the hint & context in the content we forward to Bugzilla - that's the best pointer to code we have & we could easily search for the hint/context string to find where the error comes from in our source code.

Also I guess some DBus error messages might be the same, but might have a different context/hint, so without those being included we might not be able to tell them apart ?

Reply:
Added ` webui: Add error context to Bugzilla report summary and detail `